README.md

```diff
@@ -248,12 +248,89 @@ at [alfresco-java-event-api](alfresco-java-event-api).
 ```java
     CommentsApi commentsApi;
 
-public void addComentToNode(final String nodeId){
+public void addComentToNode(final String nodeId) {
   CommentBody commentBody=new CommentBody().content("I like this file");
   commentsApi.createComment(nodeId,commentBody);
   }
-  }
+}
 ```
 
 You can find more information about how to consume the REST API
 at [alfresco-java-rest-api](alfresco-java-rest-api).
+
+#### Integrating Alfresco Event Gateway
+
+Alfresco Java SDK is now compatible with [Alfresco Event Gateway](https://www.github.com/Alfresco/alfresco-event-gateway).
+
+Using [Alfresco Event Gateway REST API](alfresco-java-rest-api/alfresco-java-rest-api-lib/alfresco-event-gateway-api), extensions can manage the lifecycle of an event subscription.
+
+For example, an out-of-process extension may create a subscription to receive certain types of events in a specific topic of an ActiveMQ broker.
+
+```java
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.alfresco.gateway.handler.SubscriptionsApi;
+import org.alfresco.gateway.model.Filter;
+import org.alfresco.gateway.model.Subscription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Sample {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(Sample.class);
+
+
+  public static void create(String[] args) {
+
+    @Inject
+    SubscriptionsApi subscriptionsApi;
+
+    Map<String, String> config = new HashMap<>();
+    config.put("broker-id", "my-broker"); // Id of the a broker in alfresco-event-gateway configuration
+    config.put("destination", "topic:sample-topic"); // Name of the topic to which the gateway shall publish the events
+
+    Filter filter = new Filter();
+    filter.setType("event-type");
+
```

Comment:
To complete the filter configuration you'd need to add something like this:

```
Map<String, String> filterConfig = new HashMap<>();
filterConfig.put("event-types", "org.alfresco.event.node.Created,org.alfresco.event.node.Updated"); // Comma-separated list of event types accepted by the filter

filter.setConfig(filterConfig);
```

Reply:
Thank you!